### PR TITLE
Fix bug in iris_hitl.sdf

### PR DESCRIPTION
**Pull request by rocketengineer1982**
Fixed a bug that prevents iris_hitl from flying.  See https://github.com/PX4/PX4-SITL_gazebo/issues/768#issue-912062536.

### CODE REVIEW

File: models/iris_hitl/iris_hitl.sdf [hunk 1]
```diff
@@ -503,7 +464,7 @@
       <send_vision_estimation>0</send_vision_estimation>
       <send_odometry>1</send_odometry>
       <enable_lockstep>0</enable_lockstep>
-      <use_tcp>0</use_tcp>
+      <use_tcp>1</use_tcp>
```
Comment (Jaeyoung-Lim) [inline models/iris_hitl/iris_hitl.sdf:467]: Is this necessary?

File: models/iris_hitl/iris_hitl.sdf [hunk 2]
```diff
@@ -487,10 +447,11 @@
       <robotNamespace/>
       <imuSubTopic>/imu</imuSubTopic>
       <magSubTopic>/mag</magSubTopic>
+      <gpsSubTopic>/gps0</gpsSubTopic>
```
Comment (Jaeyoung-Lim) [inline models/iris_hitl/iris_hitl.sdf:450]: I don't think you need this tag. Can you try removing this?

File: models/iris_hitl/iris_hitl.sdf [hunk 3]
```diff
@@ -487,10 +447,11 @@
...
       <baroSubTopic>/baro</baroSubTopic>
       <mavlink_addr>INADDR_ANY</mavlink_addr>
-      <mavlink_tcp_port>4560</mavlink_tcp_port>
       <mavlink_udp_port>14560</mavlink_udp_port>
+      <mavlink_tcp_port>4560</mavlink_tcp_port>
```
Comment (Jaeyoung-Lim) [inline models/iris_hitl/iris_hitl.sdf:454]: Since you are using HITL, you are not using `mavlink_tcp_port`

File: models/iris_hitl/iris_hitl.sdf [hunk 4]
```diff
@@ -418,53 +417,14 @@
...
-    </model>
+    <include>
+      <uri>model://gps</uri>
+      <pose>0 0 0 0 0 0</pose>
+      <name>gps0</name>
+    </include>
     <joint name='gps0_joint' type='fixed'>
-      <parent>base_link</parent>
       <child>gps0::link</child>
+      <parent>base_link</parent>
```
Comment (Jaeyoung-Lim) [inline models/iris_hitl/iris_hitl.sdf:427]: unnecessary change?

File: models/iris_hitl/iris_hitl.sdf [hunk 5]
```diff
@@ -1,6 +1,5 @@
-<!-- Generated from iris.sdf.jinja for use with HITL -->
-<sdf version='1.6'>
-  <model name='iris'>
+<sdf version='1.5'>
```
Comment (Jaeyoung-Lim) [inline models/iris_hitl/iris_hitl.sdf:1]: Is there a specific reason you had to lower the sdf version?

File: models/iris_hitl/iris_hitl.sdf [hunk 6]
```diff
@@ -82,7 +82,6 @@
           <spring_reference>0</spring_reference>
           <spring_stiffness>0</spring_stiffness>
         </dynamics>
-        <use_parent_model_frame>1</use_parent_model_frame>
```
Comment (Jaeyoung-Lim) [inline models/iris_hitl/iris_hitl.sdf:85]: Could you explain why this was necessary?

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]
Thanks for the fix!

I had a few questions along the lines

**Comment by Jaeyoung-Lim** [comment]
@rocketengineer1982 Any updates?

**Comment by julianoes** [comment]
It's not clear to me what this should fix and how. And there was no follow up, so I'm closing it.

**Pull request closed without merging**